azext_edge/tests/edge/init/test_work_unit.py

```diff
@@ -628,17 +627,16 @@ def test_work_order(
         assert mocked_prepare_keyvault_access_policy.call_args.kwargs["sp_record"]
 
         mocked_prepare_keyvault_secret.assert_called_once()
+        expected_vault_uri = f"https://localhost/{keyvault_resource_id}/vault"
+
         assert mocked_prepare_keyvault_secret.call_args.kwargs["cmd"]
         assert mocked_prepare_keyvault_secret.call_args.kwargs["deployment_name"]
+        assert mocked_prepare_keyvault_secret.call_args.kwargs["vault_uri"] == expected_vault_uri
```

Comment:
you could use a variable or assert_called_once_with() to shorten things

Reply:
Made change thanks